assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -121,14 +122,104 @@ class ImageCarouselHandler extends elementorModules.frontend.handlers.Base {
 					this.swiper.autoplay.start();
 				},
 			} );
+		} else {
+			this.elements.$carousel.off( 'mouseenter mouseleave' );
 		}
 	}
 
+	updateSwiperOption( propertyName ) {
+		const elementSettings = this.getElementSettings(),
+			newSettingValue = elementSettings[ propertyName ],
+			changeableProperties = this.getChangeableProperties();
+
+		let propertyToUpdate = changeableProperties[ propertyName ],
+			valueToUpdate = newSettingValue;
+
+		// Handle special cases where the value to update is not the value that the Swiper library accepts
+		switch ( propertyName ) {
+			case 'image_spacing_custom':
+				valueToUpdate = newSettingValue.size || 0;
+				break;
+			case 'autoplay':
+				if ( newSettingValue ) {
+					valueToUpdate = {
+						delay: elementSettings.autoplay_speed,
+						disableOnInteraction: 'yes' === elementSettings.pause_on_interaction,
+					};
+				} else {
+					valueToUpdate = false;
+				}
+				break;
+			case 'autoplay_speed':
+				propertyToUpdate = 'autoplay';
+
+				valueToUpdate = {
+					delay: newSettingValue,
+					disableOnInteraction: 'yes' === elementSettings.pause_on_interaction,
+				};
+				break;
+			case 'pause_on_hover':
+				this.togglePauseOnHover( 'yes' === newSettingValue );
+				break;
+			case 'pause_on_interaction':
+				valueToUpdate = 'yes' === newSettingValue;
+				break;
+		}
+
+		// 'pause_on_hover' is implemented by the handler with event listeners, not the Swiper library
+		if ( 'pause_on_hover' !== propertyName ) {
+			this.swiper.params[ propertyToUpdate ] = valueToUpdate;
+		}
+
+		this.swiper.update();
+	}
+
+	/*updateResponsiveSwiperOption( propertyName, currentDeviceMode ) {
```

Comment:
What about responsive handling?

Reply:
It doesn't work. `swiper.update()` doesn't update responsive values, I think it's a Swiper bug, I opened an issue in the Swiper github: https://github.com/nolimits4web/swiper/issues/3539

Reply:
I left all responsive controls to re-render from PHP for now

Reply:
Then remove all the breakpoints related and commented-out code. You can keep it in a stash for future use